src/storage.rs

```diff
@@ -240,6 +240,7 @@ impl<'a, T, A, D> Join for &'a mut Storage<T, A, D> where
 
 
 #[cfg(feature="ticket")]
+/// 
```

Comment:
Suggestions for documentation are welcome! Didn't know how to properly describe the use-case /: